Formula/convox.rb

```diff
@@ -1,8 +1,8 @@
 class Convox < Formula
-  desc "Command-line interface for the Rack PaaS on AWS"
+  desc "Command-line interface for the Convox PaaS"
   homepage "https://convox.com/"
-  url "https://github.com/convox/rack/archive/20200129210159.tar.gz"
-  sha256 "945d6237ea0108ef0cd5ab9e6dae0f898710ca84a65d8480b8466e4c40ff99af"
+  url "https://github.com/convox/convox/archive/3.0.0.tar.gz"
```

Comment:
Since this is a very different versioning scheme you should add a line `version_scheme 1`